github/authorizations_test.go

```diff
@@ -357,3 +357,20 @@ func TestAuthorizationsService_DeleteImpersonation(t *testing.T) {
 		t.Errorf("Authorizations.DeleteImpersonation returned error: %+v", err)
 	}
 }
+
+func TestActivityService_RevokedAppAuthorizationEvent(t *testing.T) {
+	raw := []byte(`{"type": "RevokedAppAuthorizationEvent","payload":{"action": "revoked"}}`)
+	var event *Event
+	if err := json.Unmarshal(raw, &event); err != nil {
+		t.Fatalf("Unmarshal Event returned error: %v", err)
+	}
+
+	want := map[string]interface{}{"action": "revoked"}
```

Comment:
Can you please make `want` be the actual type (`&AppAuthorizationEvent{}`) and the cast `got` also to that type before doing a `reflect.DeepEqual`?

I think that would be much clearer to anyone reading the code what is expected... and if the cast fails, then we have also successfully tested the `ParsePayload`.

Reply:
I'm having trouble writing the code to cast this value because of the value being `[]*events`. Is there something straightforward that I'm missing @gmlewis ? Thank you!